platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/event/linking/InOrderLinkerTests.java

```diff
@@ -541,31 +445,26 @@ void selfParentTimeCreatedMismatch(final boolean useBirthRoundForAncient) {
 
     @ParameterizedTest
     @ValueSource(booleans = {true, false})
-    @DisplayName("Other parent with mismatched time created should be linked")
+    @DisplayName("Other parent with mismatched time created SHOULD be linked")
```

Comment:
I think the displayName vs javadoc consideration we discussed offline applies here. As you're modifying the long name now, probably worth changing this to a javadoc, but only if you have other revisions to make to this PR. Otherwise, it can wait.

Reply:
@kfa-aguda This is a good point, which I'll bear in mind moving forward.

For now, as suggested, I'll merge without fixing, since no other revisions need to be made on this PR.